extensions/dicom-seg/src/components/SegmentationPanel.js

```diff
@@ -0,0 +1,552 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import cornerstoneTools from 'cornerstone-tools';
+import cornerstone from 'cornerstone-core';
+import moment from 'moment';
+import Select from 'react-select';
+
+import { utils } from '@ohif/core';
+import {
+  Range,
+  ScrollableArea,
+  TableList,
+  TableListItem,
+  Icon,
+} from '@ohif/ui';
+
+import './SegmentationPanel.css';
+
+const { studyMetadataManager } = utils;
+
+const segmentationModule = cornerstoneTools.getModule('segmentation');
+const DEFAULT_BRUSH_RADIUS = segmentationModule.getters.radius || 10;
+
+/**
+ * SegmentationPanel component
+ *
+ * @param {Object} props
+ * @param {Array} props.studies
+ * @param {Array} props.viewports - viewportSpecificData
+ * @param {number} props.activeIndex - activeViewportIndex
+ * @returns component
+ */
+const SegmentationPanel = ({ studies, viewports, activeIndex }) => {
+  /* TODO: This technically defaults to 10 if undefined (bug?) */
```

Comment:
This is because it is initialised as 10 in `cornerstoneTools`. It seems you've noticed this above though, so is this TODO out of date?